pages/stands/index.tsx

```diff
@@ -18,53 +19,77 @@ const Index = ({
   props,
 }: WithDataAndLayoutProps<stands_QueryResponse>): JSX.Element => {
   const [time, setTime] = useState(Date.now());
+  const [collaboratorsId, setCollaboratorsId] = useState<string[]>([]);
+  const [currentDayCompanyIds, setCurrentDayCompanyIds] = useState<string[]>(
+    []
+  );
```

Comment:
This doesn't need to be stored in state. You could just filter out straight from the props as the state management is done by relay.

Settings this state in `useEffect` also causes unnecessary updates to the component and makes SSR obsolete.

Reply:
Good points, I've refactored it now